### PR TITLE
Update dbt docs

### PR DESCRIPTION
**Pull request by trucklos**
- Fix a bug in our markdown documentation.
- Add a more opinionated set of steps in the overview of the dbt docs.
- Fix glean-defaults documentation.
- Adds some additional notes, pretty rough, but good to get the info in there

Still could use a lot of improvements!

### CODE REVIEW

**Review by davidsonsarah — APPROVED** [review]
lgtm!

File: pages/docs/data-ops/dbt-integration.mdx [hunk 1]
```diff
@@ -1,33 +1,112 @@
+import { Steps } from 'nextra-theme-docs'
+
 # dbt Integration
 
 ## Overview
 
-Glean can import models directly from dbt. This is useful for incorporating Glean into existing DataOps workflows, allowing you to make changes to your dbt schema while identifying issues in downstream visualizations and dashboards.
+Glean can import models directly from dbt by specifying metrics in your dbt yaml file. Glean will automatically sync dbt documentation and keep metrics in sync with downstream dashboards (whether they are checked into code or not).  It's also possible to integrate with CI/CD processes.
```
Comment (meyer-glean) [inline pages/docs/data-ops/dbt-integration.mdx:7]:
```suggestion
Glean can import models directly from dbt by specifying metrics in your dbt yaml file. Glean will automatically sync dbt documentation and keep metrics in sync with downstream dashboards (whether they are checked into version control or not). The Glean dbt integration can also be integrated in CI/CD workflows.
```

File: pages/docs/data-ops/dbt-integration.mdx [hunk 2]
```diff
@@ -1,33 +1,112 @@
...
+You can use the `glean-defaults` key to specify the glean version (which is required) and connection information.
+
+```yaml filename="dbt_project.yml" copy
+
+models:
+  +meta:
+    # glean-defaults settings will be merged into each Glean model
+    glean-defaults:
+      glean: "1.0"
+      # preventUpdatesFromUI defaults to false, but can be set to true
```
Comment (meyer-glean) [inline pages/docs/data-ops/dbt-integration.mdx:22]: `preventUpdatesFromUI` defaults to `true`.

File: pages/docs/data-ops/dbt-integration.mdx [hunk 3]
```diff
@@ -1,33 +1,112 @@
...
+  - name: order_attribution
+    description: This model describes how each order is attributed to different marketing touchpoints.
+    columns:
+      - name: attribution_id
+        description: Unique ID for the attribution.
+      - name: order_id
+        description: Order ID, matches the order_id in the sales model.
+      - name: touchpoint_id
+        description: ID of the marketing touchpoint which led to the order.
+    config:
```
Comment (meyer-glean) [inline pages/docs/data-ops/dbt-integration.mdx:49]: `config` not needed

File: pages/docs/data-ops/dbt-integration.mdx [hunk 4]
```diff
@@ -1,33 +1,112 @@
...
+    # glean-defaults settings will be merged into each Glean model
+    glean-defaults:
+      glean: "1.0"
+      # preventUpdatesFromUI defaults to false, but can be set to true
+      preventUpdatesFromUI: false
+      source:
+        connectionName: Uploads # the name of the conneciton in Glean
+        # can use jinja
+        # schema is always optional
+        schema: "{{ env_var('DBT_SCHEMA') }}"
```
Comment (meyer-glean) [inline pages/docs/data-ops/dbt-integration.mdx:28]: would maybe suggest using `"{{ target.schema }}"` which might be more familiar to dbt users

**Comment by trucklos** [comment]
Merged directly

**Pull request closed without merging**